llvm/test/tools/llvm-reduce/remove-alias.ll

```diff
@@ -1,6 +1,8 @@
 ; RUN: llvm-reduce --test FileCheck --test-arg --check-prefixes=CHECK-INTERESTINGNESS --test-arg %s --test-arg --input-file %s -o %t
 ; RUN: FileCheck --check-prefixes=CHECK-FINAL --input-file=%t %s
 
+; RUN: llvm-reduce --run-delta-in-child --test FileCheck --test-arg --check-prefixes=CHECK-INTERESTINGNESS --test-arg %s --test-arg --input-file %s -o %t
```

Comment:
@rushiraj7677 our new option will not work in windows. So these tests will fail on windows machine. 
Perhaps we need to think about how we can support this in non posix environment.